### ISSUE TITLE
NumberOfCars?

### ISSUE THREAD

**Issue by Slusep** [Informatievlaanderen/OSLOthema-mobiliteitDienstregelingEnPlanning#6]
In the service
https://purl.eu/ns/mobility/timetables-and-planning#ServiceJourney
we do not support NumberOfCars Is this a wish from SNCB?

**Comment by GeertThijs** [comment]
You are correct, this is a specialisation of VehicleType that is currently missing. It consists of a Train class (with Train.numberOfCars indeed) and TrainElements. I will add this.

**Comment by GeertThijs** [comment]
Added, but still have to write the definitions.

**Comment by GeertThijs** [comment]
Definitions were written. Can be closed.
